nav_msgs/Path zone
---
nav_msgs/Path[] paths